test/nursery/autogluon_for_single_ts.py

```diff
@@ -0,0 +1,131 @@
+from autogluon import TabularPrediction as task
+from gluonts.dataset.repository.datasets import get_dataset
+from gluonts.dataset.util import to_pandas
+from pandas.tseries.holiday import USFederalHolidayCalendar as calendar
+import pandas as pd
+import matplotlib.pyplot as plt
+from gluonts.evaluation import Evaluator
+from gluonts.evaluation.backtest import make_evaluation_predictions
+from gluonts.model.forecast import SampleForecast
+from gluonts.dataset.common import ListDataset
+import json
+import numpy as np
+import matplotlib.pyplot as plt
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+class Gluonts3Auto:
+
+    """
+dataset_name:
+Name of Gluonts common dataset
+problem_type:
+Autogluon task fit problem_type
+num_samples:
+Number of samples to draw on the model when evaluating.
+"""
+
+    def __init__(self, dataset_name, problem_type, num_samples, context_length=None):
+        self.dataset = get_dataset(dataset_name, regenerate=False)
+        self.problem_type = problem_type
+        self.num_samples = num_samples
+        self.context_length = context_length if context_length else self.dataset.metadata.prediction_length
```

Comment:
Is this the same as the other module in this PR? I think this can be removed

Reply:
Do you mean the __init__? I did not see the duplication with other segments thought.

Reply:
I mean the files `src/gluonts/nursery/autogluon_for_single_ts.py` and `test/nursery/autogluon_for_single_ts.py`, they appear identical to me